index.html

```diff
@@ -0,0 +1,6 @@
+<html>
+    <head>
+    <h1>Hello Miicroverse </h1>
+    </head>
+
+</html>
```

Comment:
[REQUIRED] Please kindly make sure you understand how to work with HTML documents:

- [ ] Learn what is `<head/>` [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML)
- [ ] Learn what is `<body />` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body)

_hint: all the visible content of your  HTML documents should be presented in `<body/>` not in `<head/>`_

I strongly suggest you review all the materials provided by the current curriculum before completing this project.